release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
 
 You can now encrypt Azure storage accounts during installation by providing the installation program with a customer managed encryption key. See xref:../installing/installing_azure/installation-config-parameters-azure.adoc#installation-configuration-parameters-additional-azure_installation-config-parameters-azure[Installation configuration parameters] for descriptions of the parameters required to encrypt Azure storage accounts.
 
-[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
-==== CAPO integration into the cluster CAPI Operator (Tech Preview)
+[id="ocp-4-15-capi-openstack-tech-preview-no-upgrade"]
+==== OpenStack integration into the Cluster CAPI Operator (Tech Preview)
 
-If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifecycle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
+If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster CAPI Operator deploys the Cluster API Provider OpenStack and manages its lifecycle.
+The Cluster CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
 
-It is now possible to configure the CAPI `Machine` and CAPO `OpenStackMachine` resources in similar fashion to how Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources but not identical.
+It is now possible to configure the Cluster API `Machine` and `OpenStackMachine` resources similarly to how Machine API resources are configured.
+It is important to note that the Cluster API resources are equivalent to Machine API resources but not identical.
```

Comment:
"equivalent but not identical" is a little confusing.

Maybe "functionally equivalent but not structurally identical?" They share the same purpose, but not the same fields.

Reply:
revised, thanks!